src/service/xhr-impl.js

```diff
@@ -195,9 +176,11 @@ export class Xhr {
    *
    * @param {string} input
    * @param {?FetchInitDef=} opt_init
+   * @param {boolean=} opt_skipProcessing Disables any processing of the
+   *     response, including asserting the response is 2XX.
    * @return {!Promise<!JSONType>}
    */
-  fetchJson(input, opt_init) {
+  fetchJson(input, opt_init, opt_skipProcessing) {
```

Comment:
`#fetchJSON` and `#fetch` won't be equivalent, since they do different processing _before_ making the request (normalizing `Content-Type`, `Accept`, etc). This just make it so they both return `Response` regardless of anything else.